db_test.go

```diff
@@ -128,6 +132,7 @@ type dummyRow struct {
 	DoubleType    float64         `json:"doubleType"`
 	StringType    string          `json:"stringType"`
 	TimestampType athenaTimestamp `json:"timestampType"`
+	ArrayType     []string        `json:"arrayType"`
```

Comment:
is it always string? should we support array of other types that driver supports?

Reply:
If there are other types, I agree that we should support them. I am still trying to muddle my way through the code and figure out what is going on. The athenaType that is provided to the convertValue is just "array", so not sure how it will specify between different array types.